bokeh/charts/_data_adapter.py

```diff
@@ -137,6 +138,13 @@ def validate_values(values):
 
             return values
 
+        elif hasattr(values, '__array__'):
```

Comment:
@mrocklin considering your last comment on the related blaze issue should I go ahead to use `pd.DataFrame(np.asarray(values))` considering that "it should just work", despite `fields` or no `fields`?